### DISCUSSION TITLE
Chromosome coordinates of alternative promoters in the annotation file

### DISCUSSION THREAD

**Discussion by SemihAkincilar** [GoekeLab/proActiv#29] [Q&A] [answered]
Hi,
I read your Cell article (A Pan-cancer Transcriptome Analysis Reveals Pervasive Regulation through Alternative Promoters) and it is a fantastic paper and a resource for the scientific community. 
We analysed some of our data based on the method section in the paper. However, in the promoter annotation file, I realized that start and end coordinates are the same for alternative promoters indicated in sup. file mmc2 . 

Appreciate it if you can confirm if I am looking at the correct files.

Thanks in advance.
Semih

**Comment by jonathangoeke** [comment, marked as answer]
Hi Semih,
Thank you! The promoters are defined based on the transcription start site coordinate  which is just a single base. You can extend these coordinates by your own definition (considering the strand of the transcripts).
Best
Jonathan
